.github/workflows/build_and_test_ansi.yml

```diff
@@ -17,18 +17,18 @@
 # under the License.
 #
 
-name: ANSI SQL mode test
+name: "Build and test (ANSI)"
```

Comment:
Hmm, will we be able to remove the "ANSI SQL mode test" in the actions page?
<img width="596" alt="image" src="https://user-images.githubusercontent.com/1097932/159200340-fdf8d7d7-dc21-477b-bde9-aa33eec4a834.png">

Reply:
Yeah, we can actually manually remove